### ISSUE TITLE
MKV Resume Doesnt Appear to work on web client

### ISSUE THREAD

**Issue by matthewstrasiotto** [jellyfin/jellyfin#7745]
If I had to guess, I'd say the issue appears here:

```
[16:36:33] [INF] [65] Jellyfin.Api.Helpers.MediaInfoHelper: StreamBuilder.BuildVideoItem( Profile=Anonymous Profile, Path=/path/to/file.mkv, AudioStreamIndex=0, SubtitleStreamIndex=2 ) => ( PlayMethod=Transcode, TranscodeReason=VideoCodecNotSupported ) media:/videos/07c8704a-6112-b6a1-5bab-96ff7b600b9d/master.m3u8?MediaSourceId=07c8704a6112b6a15bab96ff7b600b9d&VideoCodec=h264&AudioCodec=aac,mp3&AudioStreamIndex=0&VideoBitrate=139616000&AudioBitrate=384000&MaxFramerate=23.976025&MaxWidth=1727&api_key=<token>&TranscodingMaxAudioChannels=2&RequireAvc=false&Tag=e2116a8b563a6fe3b58feb30c6e059c5&SegmentContainer=ts&MinSegments=1&BreakOnNonKeyFrames=True&h264-level=41&h264-videobitdepth=8&h264-profile=high&TranscodeReasons=VideoCodecNotSupported
[16:36:33] [INF] [65] Jellyfin.Api.Controllers.DynamicHlsController: Current HLS implementation doesn't support non-keyframe breaks but one is requested, ignoring that request
```

For context - Other clients (Android ExoPlayer) can resume a paused movie, and on the Item Details page, the progressbar correctly reports where playback left off

![image](https://user-images.githubusercontent.com/39424834/168415715-f6f5afe3-e536-49f5-8580-45e1b0b8b9d4.png)

**Media Info of the file**

```
file
Container: mkv
Path: /path/to/file.mkv
Size: 893 MB

Audio
Title: Eng - DTS - Stereo - Default
Language: eng
Codec: DTS
Profile: DTS
Layout: stereo
Channels: 2 ch
Bitrate: 1536 kbps
Sample rate: 48000 Hz
Default: Yes
Forced: No
External: No

Video
Title: 720p H264 SDR
Codec: H264
AVC: No
Profile: High
Level: 41
Resolution: 1280x720
Aspect ratio: 16:9
Interlaced: No
Framerate: 23.976025
Bitrate: 5718 kbps
Bit depth: 8 bit
Video range: SDR
Color space: bt709
Pixel format: yuv420p
Ref frames: 1
NAL: 0

Subtitle
Title: Eng - SUBRIP - External
Language: eng
Codec: SUBRIP
Default: No
Forced: No
External: Yes
```

**Logs**

```
Attaching to media_jellyfin_1
[16:36:04] [WRN] [85] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from https://example.com/Sessions/Playing/Stopped to 192.168.0.1 in 0:00:01.6756001 with Status Code 204
[16:36:14] [INF] [105] Jellyfin.Api.Helpers.MediaInfoHelper: User policy for Matt. EnablePlaybackRemuxing: True EnableVideoPlaybackTranscoding: True EnableAudioPlaybackTranscoding: True
[16:36:14] [INF] [105] Jellyfin.Api.Helpers.MediaInfoHelper: StreamBuilder.BuildVideoItem( Profile=Anonymous Profile, Path=/path/to/file.mkv, AudioStreamIndex=0, SubtitleStreamIndex=2 ) => ( PlayMethod=Transcode, TranscodeReason=VideoCodecNotSupported ) media:/videos/07c8704a-6112-b6a1-5bab-96ff7b600b9d/master.m3u8?MediaSourceId=07c8704a6112b6a15bab96ff7b600b9d&VideoCodec=h264&AudioCodec=aac,mp3&AudioStreamIndex=0&VideoBitrate=139616000&AudioBitrate=384000&MaxFramerate=23.976025&MaxWidth=1727&api_key=<token>&TranscodingMaxAudioChannels=2&RequireAvc=false&Tag=e2116a8b563a6fe3b58feb30c6e059c5&SegmentContainer=ts&MinSegments=1&BreakOnNonKeyFrames=True&h264-level=41&h264-videobitdepth=8&h264-profile=high&TranscodeReasons=VideoCodecNotSupported
[16:36:14] [INF] [85] Jellyfin.Api.Controllers.DynamicHlsController: Current HLS implementation doesn't support non-keyframe breaks but one is requested, ignoring that request
[16:36:14] [INF] [85] Jellyfin.Api.Helpers.TranscodingJobHelper: /usr/lib/jellyfin-ffmpeg/ffmpeg -fflags +genpts  -i file:"/path/to/file.mkv" -map_metadata -1 -map_chapters -1 -threads 0 -map 0:1 -map 0:0 -map -0:s -codec:v:0 copy -start_at_zero -codec:a:0 aac -ac 2 -ab 384000 -copyts -avoid_negative_ts disabled -max_muxing_queue_size 2048 -f hls -max_delay 5000000 -hls_time 6 -hls_segment_type mpegts -start_number 0 -hls_segment_filename "/config/data/transcodes/94b85a17c50e162fd16a1e273379c82b%d.ts" -hls_playlist_type vod -hls_list_size 0 -y "/config/data/transcodes/94b85a17c50e162fd16a1e273379c82b.m3u8"
[16:36:30] [INF] [65] Jellyfin.Api.Helpers.TranscodingJobHelper: Stopping ffmpeg process with q command for /config/data/transcodes/94b85a17c50e162fd16a1e273379c82b.m3u8
[16:36:30] [INF] [65] Jellyfin.Api.Helpers.TranscodingJobHelper: FFmpeg exited with code 0
[16:36:30] [INF] [65] Jellyfin.Api.Helpers.TranscodingJobHelper: Deleting partial stream file(s) /config/data/transcodes/94b85a17c50e162fd16a1e273379c82b.m3u8
[16:36:31] [INF] [65] Emby.Server.Implementations.Session.SessionManager: Playback stopped reported by app Jellyfin Web 10.8.0 playing Happy New Year. Stopped at 136323 ms
[16:36:31] [WRN] [65] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from https://example.com/Sessions/Playing/Stopped to 192.168.0.1 in 0:00:01.6427885 with Status Code 204
[16:36:33] [INF] [65] Jellyfin.Api.Helpers.MediaInfoHelper: User policy for Matt. EnablePlaybackRemuxing: True EnableVideoPlaybackTranscoding: True EnableAudioPlaybackTranscoding: True
[16:36:33] [INF] [65] Jellyfin.Api.Helpers.MediaInfoHelper: StreamBuilder.BuildVideoItem( Profile=Anonymous Profile, Path=/path/to/file.mkv, AudioStreamIndex=0, SubtitleStreamIndex=2 ) => ( PlayMethod=Transcode, TranscodeReason=VideoCodecNotSupported ) media:/videos/07c8704a-6112-b6a1-5bab-96ff7b600b9d/master.m3u8?MediaSourceId=07c8704a6112b6a15bab96ff7b600b9d&VideoCodec=h264&AudioCodec=aac,mp3&AudioStreamIndex=0&VideoBitrate=139616000&AudioBitrate=384000&MaxFramerate=23.976025&MaxWidth=1727&api_key=<token>&TranscodingMaxAudioChannels=2&RequireAvc=false&Tag=e2116a8b563a6fe3b58feb30c6e059c5&SegmentContainer=ts&MinSegments=1&BreakOnNonKeyFrames=True&h264-level=41&h264-videobitdepth=8&h264-profile=high&TranscodeReasons=VideoCodecNotSupported
[16:36:33] [INF] [65] Jellyfin.Api.Controllers.DynamicHlsController: Current HLS implementation doesn't support non-keyframe breaks but one is requested, ignoring that request
[16:36:33] [INF] [65] Jellyfin.Api.Helpers.TranscodingJobHelper: /usr/lib/jellyfin-ffmpeg/ffmpeg -fflags +genpts  -i file:"/path/to/file.mkv" -map_metadata -1 -map_chapters -1 -threads 0 -map 0:1 -map 0:0 -map -0:s -codec:v:0 copy -start_at_zero -codec:a:0 aac -ac 2 -ab 384000 -copyts -avoid_negative_ts disabled -max_muxing_queue_size 2048 -f hls -max_delay 5000000 -hls_time 6 -hls_segment_type mpegts -start_number 0 -hls_segment_filename "/config/data/transcodes/d64ecab4e31ab1ad330c65d1e86abe51%d.ts" -hls_playlist_type vod -hls_list_size 0 -y "/config/data/transcodes/d64ecab4e31ab1ad330c65d1e86abe51.m3u8"
[16:37:16] [INF] [97] Jellyfin.Api.Helpers.TranscodingJobHelper: FFmpeg exited with code 0
[16:37:45] [WRN] [147] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from https://example.com/Sessions/Playing/Progress to 192.168.0.1 in 0:00:01.0322985 with Status Code 204
[16:44:03] [INF] [63] Emby.Server.Implementations.Session.SessionWebSocketListener: Sending ForceKeepAlive message to 1 inactive WebSockets.
[16:44:15] [INF] [153] Emby.Server.Implementations.Session.SessionWebSocketListener: Sending ForceKeepAlive message to 1 inactive WebSockets.
[16:44:27] [INF] [134] Emby.Server.Implementations.Session.SessionWebSocketListener: Lost 1 WebSockets.

```

**FFmpeg Logs**

[FFmpeg.DirectStream-2022-05-14_16-36-14_07c8704a6112b6a15bab96ff7b600b9d_89a55e86.log](https://github.com/jellyfin/jellyfin/files/8692430/FFmpeg.DirectStream-2022-05-14_16-36-14_07c8704a6112b6a15bab96ff7b600b9d_89a55e86.log)
[FFmpeg.DirectStream-2022-05-14_16-36-33_07c8704a6112b6a15bab96ff7b600b9d_0b39b03e.log](https://github.com/jellyfin/jellyfin/files/8692431/FFmpeg.DirectStream-2022-05-14_16-36-33_07c8704a6112b6a15bab96ff7b600b9d_0b39b03e.log)

**Stats for Nerds Screenshots**

**Server System (please complete the following information):**
 - OS: Docker on linux
 - Jellyfin Version:  `image: jellyfin/jellyfin:10.8.0-beta1`
 - Hardware settings & device:
   - NVENC
   - I'm pretty sure no transcoding is happening, only remux 
 - Reverse proxy: traefik
 
**Client System (please complete the following information):**
 - Device: PC
 - OS: Windows 10
 - Client: Web
 - Browser (if Web client): Chrome
   - Browser Version: 100.0.4896.127
 - Client and Browser Version: [e.g. 10.3.4 and 68.0]

**Comment by matthewstrasiotto** [comment]
Btw- wasnt just MKV, also

It appears to be resolved by changing to `10.8.0-beta2`, I think https://github.com/jellyfin/jellyfin-web/pull/3535 fixed it

**Comment by syncmaster700nf** [comment]
With the previous version I used VLC as the external media player and it saved the video progress and I was able to start back at the point where I stopped watching but now the video files goes back to the biginning of the file.  Any ideas on how to fix this?
